recipe/recipe.yaml

```diff
@@ -0,0 +1,230 @@
+      #
+      # For now, we skip the following tests:
+      #  - Tests for Azure are deactivated for now (not present for MacOS, fails for Linux).
+      #  - `test_lmdb_warnings_when_reopened` also currently fails.
+      #  - `test_column_stats_string_column_minmax` fails for linux.
+      #  - `test_force_compact_symbol_list_lock_held_past_ttl` also currently fails for MacOS.
+      #  - `test_filter_numeric_isnotin_signed` is generally flaky
+      #  - for `test_write_non_timestamp_index`, see #246
+      #  - for `test_write_not_sorted_multi_index_exception`, see #247
+      #  - `test_compat` - remove after arcticdb #2194 is merged
+      #  - `test_modify_old_library_option_with_current` - there is problem with resolving the path
+      #  - `test_pandas_pickling` - https://github.com/conda-forge/arcticdb-feedstock/pull/393#issuecomment-2626884786
+      - |
+        ARCTICDB_RAND_SEED=$RANDOM pytest python/tests \
+          --verbose \
+          --reruns 5 \
+          --reruns-delay 1 \
+          -k "not Azure \
+              and not azurite \
+              and not test_lmdb_warnings_when_reopened \
+              and not test_force_compact_symbol_list_lock_held_past_ttl \
+              and not test_write_non_timestamp_index \
+              and not test_write_not_sorted_multi_index_exception \
+              and not test_map_size_bad_input \
+              and not test_map_size_parsing \
+              and not lmdb \
+              and not test_column_stats_string_column_minmax \
+              and not test_filter_numeric_isnotin_signed \
+              and not test_filter_numeric_isnotin_unsigned \
+              and not test_compat_ \
+              and not test_modify_old_library_option_with_current \
+              and not test_pandas_pickling"
+    requirements:
+      run:
+        # This environment specification must be maintained in sync with the one upstream:
+        # See: https://github.com/man-group/ArcticDB/blob/master/environment_unix.yml
+        # See `options.extras_require`: https://github.com/man-group/ArcticDB/blob/master/setup.cfg
+        - pytest
+        # We disable pyarrow because the environement is too constrained.
+        # Only one test is skipped via the patch applied above.
+        # TODO: find a way to add a condition to only exclude it for libevent 2.1.10
+        # - pyarrow
+        - pytest-cpp
+        - pytest-timeout
+        - packaging
+        - future
+        - mock
+        - boto3
+        - moto
+        - flask  # Used by moto
+        - flask-cors
+        - hypothesis <6.73
+        - azure-storage-blob
+        - azure-identity
+        - coverage
+        - asv
+        - virtualenv
+        - pymongo
+        - mock
+        - trustme
+        - psutil
+        - memray
+        - pytest-memray
+        # Extra dependencies
+        - pytest-rerunfailures
+        - nodejs  # for npm
+    files:
+      source:
+        - python/tests
```

Comment:
I think you just need to add a trailing / here.

Reply:
We might try again if we feel the v1 recipe format comes with significant benefits for this feedstock.